app/assets/javascripts/workarea/storefront/emarsys/modules/web_extend_adapter.js

```diff
@@ -9,7 +9,9 @@ WORKAREA.analytics.registerAdapter('webExtendAdapter', function () {
 
     var pushEmarsysEvent = function(event, payload) {
             if (WORKAREA.environment.isTest || _.isUndefined(window.ScarabQueue)) {
-                console.log('Emarsys is not configured or is disabled in test');
+                if (!WORKAREA.environment.isTest) {
+                    console.log('Emarsys is not configured');
```

Comment:
You could adjust the above conditional to
```
if (!WORKAREA.environment.isTest && _.isUndefined(window.ScarabQueue) {
  console.log('Emarsys is not configured');
}
```

Reply:
this is totally out of scope for this change so i'm removing it. we can address the issue with JS in the tests in a different PR.

Reply:
turns out this was masking the actual failures which I fixed